### PR TITLE
change locale url format

### PR DESCRIPTION
**Pull request by miltontony**

### CODE REVIEW

**Comment by miltontony** [comment]
@smn ready for review

File: cms/views/cms_views.py [hunk 1]
```diff
@@ -188,13 +188,17 @@ def flatpage(self):
         }
 
     @view_config(route_name='locale')
+    @view_config(route_name='locale_matched')
     def set_locale_cookie(self):
-        if self.request.GET['language']:
+        response = Response()
+        language = self.request.matchdict.get('language')
```
Comment (smn) [inline cms/views/cms_views.py]:
```python
language = self.request.matchdict.get('language') or self.request.GET['language']
```
?

**Comment by miltontony** [comment]
@smn updated

**Comment by smn** [comment]
:+1:

**Pull request merged**